ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.17.yaml

```diff
@@ -732,8 +780,8 @@ tests:
     - ref: ipi-install-heterogeneous
     - ref: openshift-e2e-test
     workflow: openshift-e2e-gcp-ovn
-- as: ocp-e2e-gcp-ovn-heterogeneous-day-0
-  interval: 24h
+- as: ocp-e2e-gcp-ovn-multi-day-0-x-ax
```

Comment:
This is the first time we see "deployment type" used according to the specification. I think it would be find to move it over to be `ocp-e2e-gcp-ovn-day-0-multi-x-ax` just to be consistent with the others.